releasenotes/notes/passthrough-git-am-arguments-23cd0b292304d648.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    ``git pw patch apply``, ``git pw bundle apply`` and ``git pw series apply``
+    will now pass any additional arugments provided through to ``git am``. For
+    example::
+
+        $ git pw patch apply 123 --signoff foo@example.com
+
```

Comment:
This usage is incorrect, --signoff does not accept an argument. Please remove the "foo@example.com" from the notes (this causes the command to fail with "file not found").

Reply:
Done.